content/millennium/r4/documents/document-reference.md

```diff
@@ -0,0 +1,111 @@
+---
+title: DocumentReference | R4 API
+---
+
+# DocumentReference
+
+* TOC
+{:toc}
+
+## Overview
+
+The DocumentReference resource is used to reference a clinical document for a patient within the health system. This resource supports returning a list of clinical documents, and a reference to retrieve a document as a pdf.
+
+The following fields are returned if valued:
+
+* [DocumentReference id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status]( http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.status){:target="_blank"}
+* [Document status](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.docStatus){:target="_blank"}
+* [Document type](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.type){:target="_blank"}
+* [Document category](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.category){:target="_blank"}
+* [Subject (Patient)](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.subject){:target="_blank"}
+* [Author](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.author){:target="_blank"}
+* [Authenticator/verifying provider](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.authenticator){:target="_blank"}
+* [Document description/title]( http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.description){:target="_blank"}
+* [Attachment ContentType, Created date/time, Title, and URL (fully qualified link to the document)](http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.content.attachment){:target="_blank"}
+* [Patient encounter]( http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.context.encounter){:target="_blank"}
+* [Document period]( http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.context.period){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:document_reference, :r4) %>
+
+## Search
+
+Search for DocumentReferences that meet supplied query parameters:
+
+    GET /DocumentReference?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name                     | Required?          | Type          | Description
+--------------------------|--------------------|---------------|----------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
```

Comment:
prefer to have examples for all params

Reply:
[Added missing example](https://github.com/cerner/fhir.cerner.com/pull/334/commits/5e38b77a2b8751100b9950d56d54324d1e102e2b#diff-6e001351a4f222ecfb0d8c7f4e55c2d5R51)